app/views/layouts/_top_header.html.haml

```diff
@@ -0,0 +1,46 @@
+.service-header
+  .div-nav
+    %nav.navbar.navbar-static-top.navbar-expand-lg.navbar-light
+      %div
+        = image_tag('coffee.jpg', class: "d-inline-block align-top logo-img")
+        = link_to home_path, id: 'logo-text', class: 'navbar-brand texty' do
+          EUREKA
+          %br/
+          CAFFEE
+      %button.navbar-toggler{ 'aria-controls' => 'navbar-nav', 'aria-expanded' => 'false', 'aria-label' => 'Toggle navigation', 'data-target' => '#navbar-nav', 'data-toggle' => 'collapse', :type => 'button' }
+        %span.navbar-toggler-icon
+      #navbar-nav.collapse.navbar-collapse.bg-custom
+        %ul#menu-top.navbar-nav.ml-auto
+          %li.nav-item.active
+            = link_to home_path, class: 'nav-link texty' do
+              = content_tag(:span, '·', class: 'middot')
+              HOME
+          - if logged_in?
+            %li.nav-item.dropdown
+              %a.dropdown-toggle.texty.nav-link{ 'data-toggle' => 'dropdown', :href => '#' }
+                Account
+              %ul.dropdown-menu.texty
+                %li= link_to 'Profile', current_user
+                %li= link_to 'Settings', edit_user_path(current_user)
+                %li
+                  = link_to 'Log out', logout_path, method: :delete
+            - if current_user.admin? && !current_user?(@user)
+              %li.nav-item.active
+                = link_to home_path, class: 'nav-link texty' do
+                  = content_tag(:span, '·', class: 'middot')
```

Comment:
why not a regular span?
`%span.middot Dashboard` it's easier to read. And I told you this in the previous PR

Reply:
solved. ✓.